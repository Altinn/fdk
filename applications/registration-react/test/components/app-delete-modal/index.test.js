import React from 'react';
import { shallow } from 'enzyme';
import AppDeleteModal from '../../../src/components/app-delete-modal';

test('should render AppDeleteModal correctly', () => {
  const wrapper = shallow(<AppDeleteModal />);
  expect(wrapper).toMatchSnapshot();
});
