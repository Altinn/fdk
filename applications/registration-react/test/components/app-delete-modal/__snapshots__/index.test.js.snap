// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render AppDeleteModal correctly 1`] = `
<div>
  <Modal
    autoFocus={true}
    backdrop={true}
    backdropTransition={
      Object {
        "mountOnEnter": true,
        "timeout": 150,
      }
    }
    fade={true}
    isOpen={false}
    keyboard={true}
    modalTransition={
      Object {
        "timeout": 300,
      }
    }
    onClosed={[Function]}
    onOpened={[Function]}
    role="dialog"
    toggle={[Function]}
    zIndex={1050}
  >
    <ModalHeader
      closeAriaLabel="Close"
      tag="h4"
      wrapTag="div"
    />
    <ModalBody
      tag="div"
    />
    <ModalFooter
      className="d-flex justify-content-center"
      tag="div"
    >
      <Button
        color="danger"
        onClick={[Function]}
        tag="button"
      >
        Ja, slett datasettet
      </Button>
      <Button
        color="primary"
        tag="button"
      >
        Nei, behold datasettet
      </Button>
    </ModalFooter>
    <ModalFooter
      tag="div"
    />
  </Modal>
</div>
`;
